CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-1.3.0...master
 
+### Added
+- `RdsDns`, `AuroraCluster`, `AuroraEndpoint`, `AuroraReplicasDiscoverer`
```

Comment:
Do we need to expose all the classes in API?

Reply:
Can we re-user common interfaces like `javax.sql.DataSource` instead of creating new ones?

I'd imagine we have an AuroraCluster and we can ask It for a main data source and replicaDataSources.

I don't understand the need for RdsDns, AuroraEndpoint and AuroraReplicasDiscoverer.

Reply:
Good question, it was tempted by this... All these classes are needed for com.atlassian.db.replica.api.aurora.AuroraEndpoints#instanceEndpoint
We can strip the abstraction/make it private. I will open another PR for this so we can compare.

Reply:
@wyrzyk are we good with keeping data classes package-private instead of making them public?
We could inline, but I'm worried it will make stuff less readable.